kaldb/src/main/proto/kaldb_search.proto

```diff
@@ -238,21 +240,9 @@ message SchemaResult {
   // Map of field name, to field definition
   map<string, SchemaDefinition> field_definition = 1;
 }
-
-enum FieldType {
-  TEXT = 0;
-  STRING = 1;
-  INTEGER = 2;
-  LONG = 3;
-  FLOAT = 4;
-  BOOLEAN = 5;
-  DOUBLE = 6;
-  ID = 7;
-}
-
 message SchemaDefinition {
   // Field type as defined by KalDB
-  FieldType type = 1;
+  slack.proto.kaldb.schema.SchemaFieldType type = 1;
```

Comment:
This wouldn't be backward compatible, because the enum values have changed. For example ID used to be "7" in the previous option, but would now be "14".

Reply:
It wouldn't. But we don't use it for inter node communiction. This is just a proto and we always translate the JSON back to the HTTP API. So we don't technically need to be back compat